src/main/java/org/jabref/gui/customentrytypes/CustomizeEntryAction.java

```diff
@@ -13,7 +13,7 @@ public CustomizeEntryAction(JabRefFrame frame) {
 
     @Override
     public void execute() {
-        EntryTypeCustomizationDialog dialog = new EntryTypeCustomizationDialog();
+        CustomizeEntryTypeDialogView dialog = new CustomizeEntryTypeDialogView(this.frame.getCurrentBasePanel().getBibDatabaseContext());
```

Comment:
Please use `statemanager` instead of `JabRefFrame`. Moreover, bind `executable` correctly as in e.g https://github.com/JabRef/jabref/blob/251c44d37207663834660e9582cb36c9fcedaabe/src/main/java/org/jabref/gui/metadata/BibtexStringEditorAction.java